src/components/Modal/Modal.jsx

```diff
@@ -1,7 +1,105 @@
 import React from 'react';
+import Form from './Form';
+import styled, { keyframes } from 'styled-components';
 
-function Modal() {
-  return <div></div>;
+function Modal({ handleModal, onClickWindow, isSignUp, onClickSignUp }) {
+  return (
+    <Container onClick={e => onClickWindow(e)}>
```

Comment:
```suggestion
    <Container onClick={onClickWindow}>
```
함수를 그대로 넘겨줘도 같은 결과일거 같아요